src/controllers/dlgprefcontrollerdlg.ui

```diff
@@ -100,114 +100,422 @@
      </property>
     </widget>
    </item>
+
    <item row="2" column="0">
-    <widget class="QGroupBox" name="groupBoxDeviceInfo">
+    <widget class="QWidget" name="deviceInfoGroupLayoutWidget">
      <property name="sizePolicy">
-      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
+      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
-     <property name="title">
-      <string>Device Info</string>
-     </property>
-     <property name="alignment">
-      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
-     </property>
-     <layout class="QGridLayout" name="gridLayoutDeviceInfo" rowminimumheight="1,1,1,1,1,1,1,1,1,1,0">
-      <property name="sizeConstraint">
-       <enum>QLayout::SetMaximumSize</enum>
-      </property>
-      <item row="0" column="0">
-       <widget class="QLabel" name="labelPhysicalInterface">
-        <property name="sizePolicy">
-         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
-          <horstretch>0</horstretch>
-          <verstretch>0</verstretch>
-         </sizepolicy>
-        </property>
-        <property name="text">
-         <string>Physical Interface:</string>
-        </property>
-        <property name="alignment">
-         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
-        </property>
-       </widget>
-      </item>
-      <item row="0" column="1">
-       <widget class="QLabel" name="labelPhysicalInterfaceValue">
-        <property name="sizePolicy">
-         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
-          <horstretch>0</horstretch>
-          <verstretch>0</verstretch>
-         </sizepolicy>
-        </property>
-        <property name="minimumSize">
-         <size>
-          <width>0</width>
-          <height>0</height>
-         </size>
-        </property>
-        <property name="text">
-         <string notr="true">(USB|Bluetooth|...)</string>
-        </property>
-       </widget>
-      </item>
-      <item row="1" column="0">
-       <widget class="QLabel" name="labelDataHandlingProtocol">
-        <property name="sizePolicy">
-         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
-          <horstretch>0</horstretch>
-          <verstretch>0</verstretch>
-         </sizepolicy>
-        </property>
-        <property name="text">
-         <string>Data protocol:</string>
-        </property>
-        <property name="alignment">
-         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
-        </property>
-       </widget>
-      </item>
-      <item row="1" column="1">
-       <widget class="QLabel" name="labelDataHandlingProtocolValue">
+     <layout class="QVBoxLayout" name="deviceInfoGroupLayout">
+      <item>
+       <widget class="WCollapsibleGroupBox" name="groupBoxDeviceInfo">
```

Comment:
Please ensure that groupBoxDeviceInfo and groupBoxMappingInfoare always expanded, if no mapping is loaded to a device, as these boxes contain the information needed by the end user to select the correct mapping for the right device.

Reply:
[By default these groups are expanded and remain so until the user explicitly collapses them.](https://www.deepl.com/translator)
And the most important info is the device name which is the big label in the top left corner.
What other info do you refer to?

Reply:
The text in the big label is not unique, this is why Device Info was implemented for 2.6. On my system I have 3 devices with exactly the same string:
![grafik](https://github.com/user-attachments/assets/3524f627-1c99-4eab-8cda-cfc971480398)
This is normal because this string depends on assumptions that are only sometimes true. This was addressed by adding the device info box. If this information is hidden, the user has no chance to distinguish the devices.
There is no use case for collapsing these 2 boxes of valuable information needed for the mapping selection, until a mapping is loaded. This is because the tabs below are hidden at this time and enough space is available to show them.

Reply:
The use case I'm thinking about is the beginner or first time user who needs this information to distiguish devices with same or similar names. Once it's collapsed, he might have not recognise that this information is hidden here.
the collapsing makes only sense if there is content to show instead. It makes the life harder for the beginner user with no benefit for any other use case.
But as changing the default can be done at any time later let's merge this know and look later how to restore the current user experience afterwards.
An alternative to changing the collapse state itself might be a second state, which disables the collapse mode and let it behave like a normal GroubBox.